guides/common/modules/con_installing-a-server.adoc

```diff
@@ -0,0 +1,32 @@
+[id="installing-a-server_{context}"]
+= Installing a {ProjectServer}
```

Comment:
title does not match anchor and file name. IMHO it's hugely important to know if we talk about a the project platform or client platform. For Satellite, it's all RHEL, but for the community and orcharhino, it's quite different.

Reply:
I made sure the heading matches ID and file name.